WordPress/src/main/java/org/wordpress/android/networking/WPDelayedHurlStack.java

```diff
@@ -98,8 +99,10 @@ public HttpResponse performRequest(Request<?> request, Map<String, String> addit
                 additionalHeaders.put("Authorization", auth);
             }
 
+            String blogUriScheme = WordPress.currentBlog.getUri().getScheme();
```

Comment:
Hmm `mCurrentBlog` was always null in my testing. I'll investigate this further.